src/features/ticketing/ticketingTypes.ts

```diff
@@ -0,0 +1,40 @@
+export interface CartItem {
+    product_id: number,     // references state.tickets.eventid
+    qty: number,
+    name: string,
+    desc: string,
+    product_img_url: string,
+    price: number,
+}
+
+export interface Ticket {
+    eventid: number,             // references showtime.id in database
+    playid: string,
+    event_title: string,
+    admission_type: string,
+    eventdate: string,
+    starttime: string,
+    ticket_price: number,
+    concession_price: number,
+    available: number,
+}
+
+export interface Play {
+    id: string,
+    title: string,
+    description: string,
+    image_url: string,
+}
+
+export interface ticketingState {
+    cart: CartItem[],
+    tickets: Ticket[],
+    plays: Play[],
+    status: LoadStatus,
+    selection: {
```

Comment:
personally I don't think selected ticket should be stored in redux state

Reply:
The intention was to eventually make the selected showing more salient elsewhere in the UI - which is determined from the ticket.

That being said, I'm good with moving this back into the EventPage component, along with your new showing selector since it's probably not going to be reused in a different component.

Reply:
This is removed from redux and handled in Event Page component